python/pyspark/pandas/tests/test_stats.py

```diff
@@ -31,6 +31,11 @@
 from pyspark.testing.sqlutils import SQLTestUtils
 
 
+# This is used in run-tests.py to discover the slow test. See more in the doc of
+# _discover_python_unittests of dev/sparktestsupport/modules.py
+_slow_test = True
```

Comment:
sorry for a forth and back. I think we can just remove the leading underscore since this is referred in another module.

```suggestion
is_slow_test = True
```

Reply:
Sure, it make sense